base/tests/utils/unit/com/google/idea/blaze/base/scope/ErrorCollector.java

```diff
@@ -35,6 +36,14 @@ public void assertNoIssues() {
     assertThat(issues).isEmpty();
   }
 
+  public List<IssueOutput> issuesOf(IssueOutput.Category category) {
+    return issues.stream().filter(i -> i.getCategory().equals(category)).collect(ImmutableList.toImmutableList());
```

Comment:
`ImmutableList.toImmutableList()` doesn't exist until guava 21, so it won't be available in Android Studio. Use `GuavaHelper.toImmutableList()`.

Reply:
`Collectors.toList()` to avoid the extra dependency.